.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -82,6 +82,12 @@ on:
         required: false
         description: >
           The repository that hosts the code that will be swapped. This is necessary for adding tags.
+      ref:
+        type: string
+        required: false
+        description: >
+          The branch, tag or SHA to checkout. When checking out the repository that triggered a workflow, this defaults
+          to the reference or SHA for that event. Otherwise, uses the default branch.
```

Comment:
Both the name and description are really generic. It's unclear what this is for and how I as a user should configure it.

Reply:
When you use the checkout action, you're in the context of a git checkout, so having a `ref` parameter makes sense. When you use the swap workflow, you're in a much higher-level context, where simply "ref" is not clear enough. Especially that it's not the ref of the current repo but the app code one, and it works together with the `app-code-repo` input.

Naming it `app-code-repository-ref` would be appropriate (and rename the other to `app-core-repository`), but the description still needs to explain what's it for **exactly**, not in general for any repo using the checkout action, and how to configure.

Reply:
removed this based on the other conversation